aspnetcore/migration/mvc/sample/WebApp1.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Content Include="wwwroot\css\site.css" />
+    <Content Include="wwwroot\css\site.min.css" />
+    <Content Include="wwwroot\favicon.ico" />
+    <Content Include="wwwroot\js\site.js" />
+    <Content Include="wwwroot\js\site.min.js" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="BundlerMinifier.Core" Version="2.3.327" />
```

Comment:
I would suggest removing this `<PackageReference />` entry for "BundlerMinifier.Core" and using the following instead:

```xml
<DotNetCliToolReference Include="BundlerMinifier.Core" Version="2.3.327" />
```

This way, the `dotnet bundle` CLI command is usable from a script in the CSPROJ file.